### PR TITLE
Contributing updates v2

### PR DESCRIPTION
**Pull request by benoittgt**
Hello

This PR is meant [to continue the work](https://twitter.com/samphippen/status/938746864123498496) done by @samphippen on #1705. I answered to few of the comments of @ashleygwilliams and @myronmarston. 

My english is not that good. So feel free to make any comments.

Their is a related PR on rspec-core https://github.com/rspec/rspec-core/pull/2485 that could be added into `CONTRIBUTING.md` and referenced from the issue template.

On the top of `CONTRIBUTING.md` their is this text:
```
<!---
 This file was generated on 2015-12-07T22:01:06+11:00 from the rspec-dev repo.
 DO NOT modify it by hand as your changes will get lost the next time it is generated.
 -->
```
Is it still the case? Should I make this PR against : https://github.com/rspec/rspec-dev/blob/master/common_markdown_files/CONTRIBUTING.md.erb ?

Thanks in advance for the review.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,3 +23,33 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## Rspec issues labels definition
+
+### `Your first PR` issues
+
+These issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a potential meaningful
```
Comment (ebihara99999) [inline CONTRIBUTING.md]: `get started on` → `get started`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,3 +23,33 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
...
+contribution to the project that should not be too hard to pull off.
+
+### `Needs reproduction case` issues
+
+These issues are ones that have been labelled by the maintainers that we
+believe do not currently have enough information to be reproduced the RSpec
+team. While not directly counted by the GitHub contribution graph, we consider
+helping us to reproduced the issue with a repro case as an extremely meaningful
```
Comment (ebihara99999) [inline CONTRIBUTING.md]: `helping us to reproduced the issue with a repro case` would be replaced by`giving/offering the reproduction steps`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,3 +23,33 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
...
+contribution.
+
+### `Has reproduction case` issues
+
+Issues that have reproduction cases have a repository that we can clone that
```
Comment (ebihara99999) [inline CONTRIBUTING.md]: This head would be better to start by `These issues are the ones that...` because others start like that.

Comment (ebihara99999) [reply CONTRIBUTING.md]: Do you mean this part as follows?
```
These issues are the ones that have reproduction cases, able to start working on immediately. 
These are good ones to tackle to help us actively fix bugs.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (JonRowe) [reply CONTRIBUTING.md]: `get started on` is the correct english here, an alternative would be `get started with`.

File: CONTRIBUTING.md (hunk 2 above)
Comment (JonRowe) [reply CONTRIBUTING.md]: Again the english is correct here, apart from a typo, `s/reproduced/reproduce`.

File: CONTRIBUTING.md (hunk 3 above)
Comment (JonRowe) [reply CONTRIBUTING.md]: The former of your suggestions is good, I like that one

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -23,3 +23,33 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
...
+enable us to quickly determine the issue is valid and then start debugging
+within RSpec. These issues are good ones to tackle to help us actively fix bugs.
+
+## Dev environment
```
[the remark this answers is not in the document]
Comment (benoittgt) [reply CONTRIBUTING.md]: I'm sorry I have no idea for this one.

**Comment by benoittgt** [comment]
Sorry for the delay. Still one point to fix. But I have no idea how to improve.

**Comment by ebihara99999** [comment]
Thanks for the previous commit. 

Where is that? I could suggest something.

**Comment by benoittgt** [comment]
Sorry I'm on my phone. See the comment on "Dev environment". Thanks

On April 16, 2018 6:11:21 PM GMT+02:00, Yusuke Ebihara <notifications@github.com> wrote:
>Thanks for the previous commit. 
>
>Where is that? I could suggest something. 
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/rspec/rspec-rails/pull/1925#issuecomment-381660260

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (JonRowe) [reply CONTRIBUTING.md]: Just remove it for now

**Comment by benoittgt** [comment]
CI fail seems unrelated : https://travis-ci.org/rspec/rspec-rails/jobs/367774520

File: CONTRIBUTING.md (hunk 2 above)
Comment (JonRowe) [reply CONTRIBUTING.md]: @benoittgt can we get the typo here fixed up?

**Comment by benoittgt** [comment]
Oups sorry. Thanks for the rebuild. It turned green.

**Comment by JonRowe** [comment]
Thanks @benoittgt

**Pull request merged**